hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,150 @@
  */
 
 import React from 'react';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
 
 export default function Settings() {
   return (
     <>
-      <h1>Settings</h1>
+      <Tabs defaultActiveKey="signals" id="setting-tabs">
+        <Tab eventKey="signals" title="Signals">
```

Comment:
Best if you create another Component called `<SignalSettingsTab>` which extends from `<Tab>`. That way, the individual files do not get super long.